### ISSUE TITLE
Elevator Trim Wheel Input issue

### ISSUE THREAD

**Issue by zoideberg8000** [flybywiresim/a32nx#7404]
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2022-08-09T10:25:11+00:00",
    "ref": "refs/heads/experimental",
    "sha": "700c7fe4490e27184e9b129864e094e028d2c23b",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "experimental:700c7fe4"
}
```

### Describe the bug

When using the mouse wheel to move the trim wheel, rotating in either up or down direction moves trim nose down (-). The number pad key shortcuts (7 and 1) still work as expected.

### Expected behavior

When rotating the mouse wheel down (towards you) the trim should increase in the  nose up (+) direction, when moving the wheel up (away from you) the trim should move in the nose down (-) direction. All other knobs move in the expected direction.

### Steps to reproduce

Spawn at gate
Power hydraulic system
Rotate mouse wheel either direction
Observe trim wheel move in nose down direction

**Comment by AdamNZ** [comment]
I can confirm this. Dragging up/down with the mouse button held down works fine, but the scroll wheel gives odd results. Sometimes up, sometimes down - depending [also] on whether the trim wheel has just been moved with the mouse button.

**Comment by crocket63** [comment]
Mmmm never tested mouse wheel in fact. Would need @lukecologne xml magic there :D

**Comment by donstim** [comment]
I found that using the scroll wheel will always turn the trim wheel in the last direction that the trim wheel was moved.

**Comment by tracernz** [comment]
Yeah, also found the trim wheel doesn't work properly at the moment as I also use scroll wheel.

**Comment by ferryvanaesch** [comment]
Same here. I did verify that the wheel is still calibrated properly but I’m suffering from the same issue as the others above.

**Comment by ferryvanaesch** [comment]
Apologies for the unclear message: I use a steering wheel to manage the trim wheel. This worked fine until recently and I'm noticing the same issues as the people above.

**Comment by OliveBeast20441** [comment]
Also confirming this bug. Trim wheel only moves in one direction, regardless of mouse wheel scrolling up or down. The NumPad keyboard inputs work normally.

**Comment by crocket63** [comment]
> Also confirming this bug. Trim wheel only moves in one direction, regardless of mouse wheel scrolling up or down. The NumPad keyboard inputs work normally.

Status for now is the feature is not intended to work with mouse scroll wheel. Maybe someone can have a look into this.

**Comment by ferryvanaesch** [comment]
It used to work though, both with mouse wheel and joystick axis. Would be great if someone knowledgeable (i.e. not me...) could have a look at this.

**Comment by crocket63** [comment]
> It used to work though, both with mouse wheel and joystick axis. Would be great if someone knowledgeable (i.e. not me...) could have a look at this.

It worked on default msfs system, now it's a new realistic system built from scratch.

**Comment by OliveBeast20441** [comment]
Very goo. I will use the NumPad inputs to set trim from now on.

**Comment by alessandroceleste** [comment]
I can also confirm that pitch trim doesn't work as expected using external hardware (mouse scroll wheel, throttle). I double-checked that my hardware is not configured in the wrong way with Asobo A320Neo (and other MSFS2020 planes).

Further setup details:
- using left dial of _VPC MongoosT-50CM3 Throttle_ as virtual axis
- using MSFS2020 setting _Elevator Trim Axis (-100% to 100%)_
